docs/installation.md

```diff
@@ -69,7 +70,7 @@ platform. Here are a couple of examples:
 <!--hipSYCL + Ninja -->
 
 ```
-cmake -G Ninja .. -DCMAKE_PREFIX_PATH="<path-to-hipsycl-install>/lib/cmake" -DHIPSYCL_PLATFORM=cuda -DHIPSYCL_GPU_ARCH=sm_52 -DCMAKE_INSTALL_PREFIX="<install-path>" -DCMAKE_BUILD_TYPE=Release
+cmake -G Ninja .. -DCMAKE_PREFIX_PATH="<path-to-hipsycl-install>" -DHIPSYCL_PLATFORM=cuda -DHIPSYCL_GPU_ARCH=sm_52 -DCMAKE_INSTALL_PREFIX="<install-path>" -DCMAKE_BUILD_TYPE=Release
```

Comment:
`HIPSYCL_PLATFORM` + `HIPSYCL_GPU_ARCH` are deprecated, we should show `HIPSYCL_TARGETS=cuda:sm_52` here.

Reply:
Right, fixed!